http_parser.h

```diff
@@ -201,6 +201,8 @@ enum http_status
   /* RFC-2068, section 19.6.1.2 */  \
   XX(31, LINK,        LINK)         \
   XX(32, UNLINK,      UNLINK)       \
+  /* SHOUTcast */                   \
```

Comment:
Icecast clients use this method.  Legacy SHOUTcast clients use a completely incompatible protocol (sadly).  The rest of this looks awesome, thanks!

Reply:
I'll update that before I land it, here and in the commit log.